src/bitwise/core.cljs

```diff
@@ -147,7 +150,11 @@
                                  :disabled (= process-slots-available 0)} "<"]
                        [program-info program]])]]]
      [:div
-      [:h3 {:style {:margin-bottom 8}} "debug"]
+      [:h3 {:style {:margin-bottom 8}} "Developer Tools"]
+      [:div 
+        [action-button {:on-click #(swap! game-state assoc-in [:programs] (into #{} (keys program-catalog)))} "Grant All Programs"]
+        [action-button {:on-click #(reset! game-state (default-state))} "Reset Default State"]]
```

Comment:
Can you wrap this in a chicken switch? `js/confirm` shows a browser confirmation alert, and returns true or false

Reply:
Er, I should say, it delegates to `window.confirm`